test/alternator/test_metrics.py

```diff
@@ -130,6 +131,31 @@ def test_batch_write_item(test_table_s, metrics):
         test_table_s.meta.client.batch_write_item(RequestItems = {
             test_table_s.name: [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}}]})
 
+def test_batch_write_item_histogram(test_table_s, metrics):
+    with check_increases_metric_exact(metrics, "scylla_alternator_batch_item_count_histogram_bucket", [[0, {'op': 'BatchWriteItem', 'le': '2.000000'}], [1, {'op': 'BatchWriteItem', 'le': '3.000000'}]]):
+        test_table_s.meta.client.batch_write_item(RequestItems = {
+            test_table_s.name: [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}}, {'PutRequest': {'Item': {'p': random_string(), 'a': 'hi1'}}}, {'PutRequest': {'Item': {'p': random_string(), 'a': 'hi1'}}}]})
+    items = [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}} for i in range(100)]
+    items2 = [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}} for i in range(100)]
```

Comment:
Note that a batch with 100 items is an Alternator extension, DynamoDB limits BatchWriteItem to 25 items (see https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BatchWriteItem.html).
Since your intention isn't to check specifically the number 100 but just something more than 3, you could have used 25.
But it's not important.

Reply:
I wanted to see that we covered 100, with so many moving parts, it's easy to get to a situation where metrics will increase but not the right bucket or value.